gatsby-remark-oembed-md-site/src/components/Bio.js

```diff
@@ -1,21 +1,21 @@
-import React from "react";
-import { Link } from "gatsby";
+import React from 'react'
```

Comment:
In regards to the formatting, as I mentioned, `npm run format` wasn't persisting changes, but the changes here seem to be picking up a different prettier configuration as the project defaults to double quotes. Anyways, as I mentioned, I'll have a PR up to fix the formatting command, and then you can run it and things should be all good. There is work I'll be doing in the near future, to handle this on the pre-commit. See #120.